geoAssembler/geometry.py

```diff
@@ -45,7 +46,7 @@ class GeometryAssembler:
     detector_name = 'generic'
 
     def __init__(self, kd_geom):
-        """The class is instanciated using a karabo_data geometry object."""
+        """The class is instanciated using an extra_geom geometry object."""
         self.kd_geom = kd_geom
```

Comment:
Just to avoid confusion, maybe we want to change this variable name, like ex_geom or something else.

Reply:
Yep, good idea. I'll fix that, and then merge once the tests are passing.